pom.xml

```diff
@@ -724,6 +724,28 @@ https://gitbox.apache.org/repos/asf?p=maven-pmd-plugin.git;a=blob_plain;f=src/ma
           </plugins>
         </pluginManagement>
         <plugins>
+          <!-- The release artifacts don't contain our custom config files for these plugins -->
```

Comment:
in theory, many other plugins could be disabled for this, but it's annoying to maintain, so we only disable those that require custom files not part of the release tarball